covalent_queuer/tests/app/core/queuer_test.py

```diff
@@ -0,0 +1,18 @@
+import os
+
+import mock
+import nats
+import requests
+from app.core.queuer import Queue
+
+
+class TestQueuer:
+
+    MOCK_MQ_CONNECTION_URI = "localhost:4222"
+
+    @mock.patch.object(nats, "connect", autospec=True)
+    @mock.patch.dict(os.environ, {"MQ_CONNECTION_URI": MOCK_MQ_CONNECTION_URI})
+    def test_get_client(self, mocked_nats):
```

Comment:
Might need to be skipped for now until test is replaced

Reply:
Done.